server/middleware/roleBasedAccessControl.ts

```diff
@@ -1,8 +1,16 @@
 import authorisationMiddleware from './authorisationMiddleware'
 
-const hasEditAuthority = () => authorisationMiddleware(['ROLE_EDUCATION_WORK_PLAN_EDITOR'])
+enum ApplicationRoles {
+  ROLE_EDUCATION_WORK_PLAN_EDITOR = 'ROLE_EDUCATION_WORK_PLAN_EDITOR',
+  ROLE_EDUCATION_WORK_PLAN_VIEWER = 'ROLE_EDUCATION_WORK_PLAN_VIEWER',
+}
 
-const hasViewAuthority = () =>
-  authorisationMiddleware(['ROLE_EDUCATION_WORK_PLAN_EDITOR', 'ROLE_EDUCATION_WORK_PLAN_VIEWER'])
+const checkUserHasEditAuthority = () => authorisationMiddleware([ApplicationRoles.ROLE_EDUCATION_WORK_PLAN_EDITOR])
```

Comment:
I renamed these 2 exported function to `checkUserHas....` so it is consistent with our other middleware functions that check things (ie. check a form is in the session etc) before handing over to the controller